### DISCUSSION TITLE
How to interpret results

### DISCUSSION THREAD

**Discussion by Ze-213-33** [kyleskom/NBA-Machine-Learning-Sports-Betting#134] [Q&A]
Hello,
I'm a bit confused about the interpretation of the results. For simplicity let's put the 2nd part (NN model) and the over/under prediction for both models on the side.
So we are left with the predictions of who will win and lose and on the other hand the expected value now how do I place my bets based on this information:

---------------XGBoost Model Predictions---------------
Detroit Pistons vs **Charlotte Hornets (53.1%)**
**Indiana Pacers (69.1%)** vs Houston Rockets
Orlando Magic vs **Utah Jazz (57.5%)**
**Memphis Grizzlies (63.5%)** vs Golden State Warriors
**Milwaukee Bucks (63.7%)** vs Brooklyn Nets
**Sacramento Kings (56.7%)** vs New York Knicks:
--------------------Expected Value---------------------
Detroit Pistons EV: 6.0
Charlotte Hornets EV: -11.03
Indiana Pacers EV: -14.39
Houston Rockets EV: 32.68
Orlando Magic EV: -29.83
Utah Jazz EV: 32.15
Memphis Grizzlies EV: 33.37
Golden State Warriors EV: -35.44
Milwaukee Bucks EV: -29.2
Brooklyn Nets EV: 157.59
Sacramento Kings EV: -10.78
New York Knicks EV: 6.62
-------------------------------------------------------
Sometimes the expected value is very high for a team but at the same time the model says they will lose for example:
 Houston Rockets EV: 32.68 but model predict **Indiana Pacers (69.1%)** vs Houston Rockets 

Sometimes the expected value is negative for a team but at the same time the model says they will win for example:
Milwaukee Bucks EV: -29.2 but model predict **Milwaukee Bucks (63.7%)** vs Brooklyn Nets
 
What should I follow to place my bets please?

**Comment by marine1983boots** [comment]
You've missed out the information that ive been using. The over and under score. I've been placing bets using that and then cross referencing with the EV

![image](https://user-images.githubusercontent.com/127334214/224018576-764c1bbf-a5a5-40c8-8276-17c01eac4951.png)

![image](https://user-images.githubusercontent.com/127334214/224018762-4c40ad8e-4de4-40ea-a696-5a54b0c6efbb.png)

So for the pistons has positive EV. I will place the over/under on betfair. 

I could be using it completely wrong though lol Only started with bank of £20 and 74% win rate over 7 bets. Nothing at the moment. but seems to be working

**Reply by sonymag** [reply]
Friend, could u tell me, how can I understand what team will win, so I see:
Detroit Pistons vs Charlotte Hornets (53.1%): OVER 234.5 (63.7%)
is correct, that will win Charlotte Hornets(green color)? I dont understand this percent, 53.1 and 63.7, who owns it?

**Comment by chriseling** [comment]
Hornets are favored, the Over is favored

On Sun, Apr 23, 2023 at 3:48 AM @duneavv ***@***.***> wrote:

> Friend, could u tell me, how can I understand what team will win, so I see:
> Detroit Pistons vs Charlotte Hornets (53.1%): OVER 234.5 (63.7%)
> is correct, that will win? I dont understand this percent, 53.1 and 63.7,
> who owns it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/discussions/134#discussioncomment-5699324>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAVP5W7UDNNYHSUUPTHNOQ3XCUCIXANCNFSM6AAAAAAVUZ3LSA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <kyleskom/NBA-Machine-Learning-Sports-Betting/repo-discussions/134/comments/5699324
> @github.com>
>
-- 
Best,
Chris

**Reply by sonymag** [reply]
Chris, tell me pls, how did u understand Hornets are favored, with percent (63.7%) ?
Many thx friend!

**Comment by chriseling** [comment]
50% would be a coin flip or tossup, they would be favored to win 50 percent
of the time. At 63.7 percent, if they played 100 games, they would be
predicted to win 63 or 64 of those games.

On Sun, Apr 23, 2023 at 3:53 PM @duneavv ***@***.***> wrote:

> Chris, tell me pls, how did u understand Hornets are favored, with percent
> (63.7%) ?
> Many thx friend!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/discussions/134#discussioncomment-5702557>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAVP5W6KEIJI273HBDNP2BLXCWXIFANCNFSM6AAAAAAVUZ3LSA>
> .
> You are receiving this because you commented.Message ID:
> <kyleskom/NBA-Machine-Learning-Sports-Betting/repo-discussions/134/comments/5702557
> @github.com>
>
-- 
Best,
Chris
